From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Fri, 29 May 2020 16:50:41 +0800
Subject: [PATCH 2/2] gopls: use v0.2.2

Both v0.3.4 and latest (v0.4.1) are too memory hungry.  Opening a single
file in yunion.io/x/onecloud will occupy over 1.6GB of memory
---
 plugin/go.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugin/go.vim b/plugin/go.vim
index 8a7a0d9..d27bbe9 100644
--- a/plugin/go.vim
+++ b/plugin/go.vim
@@ -47,7 +47,7 @@ let s:packages = {
       \ 'godef':         ['github.com/rogpeppe/godef@master'],
       \ 'goimports':     ['golang.org/x/tools/cmd/goimports@master'],
       \ 'golint':        ['golang.org/x/lint/golint@master'],
-      \ 'gopls':         ['golang.org/x/tools/gopls@latest', {}, {'after': function('go#lsp#Restart', [])}],
+      \ 'gopls':         ['golang.org/x/tools/gopls@v0.2.2', {}, {'after': function('go#lsp#Restart', [])}],
       \ 'golangci-lint': ['github.com/golangci/golangci-lint/cmd/golangci-lint@master'],
       \ 'gomodifytags':  ['github.com/fatih/gomodifytags@master'],
       \ 'gorename':      ['golang.org/x/tools/cmd/gorename@master'],
